utils/oscap-podman

```diff
@@ -74,14 +79,18 @@ else
 fi
 DIR=$(podman mount $ID) || die
 
+for VAR in `podman inspect $ID --format '{{join .Config.Env " "}}'`; do
+    eval "export $VAR"
```

Comment:
Can a container contain an environment variable that affects the `oscap` execution?

Reply:
It was my initial concern.

In theory — it could. But I don't think it's different from environment affecting on-line scan. We are talking about situations where customer can't check JBOSS container because there is a check against JAVA_HOME variable (or something like that, can't find the bug right now).

Anyway, I can add all these VARs with some prefix (like OFFLINE_) and use this prefix in the probe. But I'm not sure if 1) this is really needed, and 2) it won't affect some content functionality (as names would be prefixed).

Reply:
The bug: https://bugzilla.redhat.com/show_bug.cgi?id=1493614

Reply:
@evgenyz Probably it is OK then, let's not do anything now.